<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitUnkEditor</class>
 <widget class="QDialog" name="SplitUnkEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit unknown split properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinUnk3">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="sliderUnk4">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinUnk2">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblUnk3">
       <property name="text">
        <string>Unknown 3</string>
       </property>
       <property name="buddy">
        <cstring>spinUnk3</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblUnk4">
       <property name="text">
        <string>Unknown 4</string>
       </property>
       <property name="buddy">
        <cstring>spinUnk4</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="sliderUnk1">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="pageStep">
        <number>1024</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinUnk1">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderUnk3">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="pageStep">
        <number>1024</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblUnk2">
       <property name="text">
        <string>Unknown 2</string>
       </property>
       <property name="buddy">
        <cstring>spinUnk2</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderUnk2">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinUnk4">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblUnk1">
       <property name="text">
        <string>Unknown 1</string>
       </property>
       <property name="buddy">
        <cstring>spinUnk1</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="lblUnk1Hex">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>0x0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="lblUnk2Hex">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>0x0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="lblUnk3Hex">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>0x0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="lblUnk4Hex">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>0x0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupUnkFlags">
     <property name="title">
      <string>Unknown flags</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkUnkFlag8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkUnkFlag7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkUnkFlag6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkUnkFlag5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkUnkFlag4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkUnkFlag3">
        <property name="text">
         <string>3 (Noise?)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SplitUnkEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SplitUnkEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderUnk1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinUnk1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinUnk1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderUnk1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderUnk2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinUnk2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinUnk2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderUnk2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderUnk3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinUnk3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinUnk3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderUnk3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderUnk4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinUnk4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinUnk4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderUnk4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
